src/storage-controller/src/lib.rs

```diff
@@ -2668,25 +2631,6 @@ where
             .collect();
 
         self.append_to_managed_collection(id, updates).await;
-
-        Some(
-            latest_row_per_id
```

Comment:
I can't comment on the above lines but you can probably remove 
https://github.com/MaterializeInc/materialize/pull/24157/files#diff-3d40535ccc11f976645c3c15ed4b6b1a08b7c04de0d21085ba2a4a891b6e5422L2624-L2631
(the code that populates `latest_row_per_id`)

Reply:
Oh, I had not realized we had both `latest_row_per_id` and `last_n_entries_per_id`––I assumed that determining the `latest_row_per_id` just worked by using `last_n_entries_per_id` and set _n_ to 1. Thanks for pointing out my oversight here.